distributedlog-core/pom.xml

```diff
@@ -52,6 +52,12 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.jmock</groupId>
```

Comment:
I realize adding a new dependency could be an issue, at least it's under the test scope.